### ISSUE TITLE
Issue regarding fileupload to rasterizer in .net

### ISSUE THREAD

**Issue by dearwebmaker** [jhabjan/Ghostscript.NET#88]
![image](https://user-images.githubusercontent.com/77912648/112323750-15b5e980-8cd8-11eb-990c-8674e3611dfa.png)

Hi Habjan,

I am trying to give input file from FILEUPLOAD.filename option in C#. But We are getting **System.IO.FileNotFoundException: 'Could not find input file.'**

**Comment by jhabjan** [comment]
You need to use absolute path like "C:\MyFiles\CDP.pdf". Ghostscript doesn't know where CDP.pdf is.

**Comment by dearwebmaker** [comment]
Actually we are giving input from FILEUPLOAD control (from UI)

**Comment by jhabjan** [comment]
Did you save that file from FileUpload control to the disk before passing it to Ghostscript? 

From what I see, you get file via FileUpload control and then just passing file's "client side" name to the Ghostscript.NET. You need to save that file to disk, get absolute path to that file and pass that absolute path Ghostcript.NET.

**Comment by dearwebmaker** [comment]
Is there any way to pass file directly from FileUpload control to Ghostcript.NET , instead of saving file to folder. ?

**Comment by jhabjan** [comment]
Yes, you can pass in byte array instead of filename and that would work but file will be saved to the disk anyway. Native Ghostscript requires random access to the file.
